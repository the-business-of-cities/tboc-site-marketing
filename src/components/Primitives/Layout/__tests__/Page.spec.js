import { Page } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Page", () => {
  it("renders correctly", () => {
    expect(shallow(<Page />)).toMatchSnapshot();
  });
});
