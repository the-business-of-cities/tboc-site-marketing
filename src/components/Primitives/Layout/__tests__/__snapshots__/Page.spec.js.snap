// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0;
  padding: 0;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media (max-width:999px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

<div
  className="c0"
/>
`;
